src/actions/assay_actions.js

```diff
@@ -53,6 +57,24 @@ function saveMolecule(saved_mols, selected_r_groups) {
   };
 }
 
+function showInvoiceSucceeded(invoice) {
```

Comment:
can you make a separate pull request on a branch that doesnt have any invoice related changes! I think its causing bugs and its difficult to tell where theyre coming from!